### DISCUSSION TITLE
ESP8266 unable to connect to WiFi

### DISCUSSION THREAD

**Discussion by OluA** [esp8266/Arduino#8101] [Q&A]
I am unable to connect to WiFi using ESP8266 wemos mini.

I have:
1. tried to compile/upload multiple sketches
2. changed between board WeMos D1 R1 and LOLIN(WEMOS) D1 R2 & mini
3. used correct and incorrect wifi ssid and password
4. modified sketch book location
5. Initially used library ESP8266 3.0.0 then changed to ESP8266 2.7.4
6. ran examples (NTP-TZ-DST/googleLocation) with correct and incorrect wifi ssid and password

I am running sketch using Arduino 1.8.7

Did all this and got either:
Attempting to connect to WPA SSID: xyz
Status = 1

Attempting to connect to WPA SSID: xyz
Status = 6

I am new to this tool, please can you tell me what is wrong

**Comment by Tech-TX** [comment]
Most likely is a Bad D1 Mini.  I have numerous D1 Mini boards that connect with no issues.  One of the WiFi example sketches is a station / SSID  scanner.  Run that and monitor the output.  If it doesn't see any SSIDs then you will never get a connection.  Virtually all D1 Mini boards are cheap Chinese clones with no quality control,  which is why I suspect the board.

**Comment by cniedzi** [comment]
> I am unable to connect to WiFi using ESP8266 wemos mini.
> 
> I have:
> 
> 1. tried to compile/upload multiple sketches
> 2. changed between board WeMos D1 R1 and LOLIN(WEMOS) D1 R2 & mini
> 3. used correct and incorrect wifi ssid and password
> 4. modified sketch book location
> 5. Initially used library ESP8266 3.0.0 then changed to ESP8266 2.7.4
> 6. ran examples (NTP-TZ-DST/googleLocation) with correct and incorrect wifi ssid and password
> 
> I am running sketch using Arduino 1.8.7
> 
> Did all this and got either:
> Attempting to connect to WPA SSID: xyz
> Status = 1
> 
> Attempting to connect to WPA SSID: xyz
> Status = 6
> 
> I am new to this tool, please can you tell me what is wrong

What router? ESP core higher then 2.5.2 doesn't work with asus routers.
